package.json

```diff
@@ -93,6 +93,7 @@
     "@cspell/dict-markdown": "^2.0.1",
     "@cspell/eslint-plugin": "workspace:*",
     "@eslint/eslintrc": "^3.0.2",
+    "@eslint/js": "^9.0.0",
```

Comment:
Is this to side-step doing the ESLint upgrade to v9 at the same time?

Reply:
Thank you for catching the mismatch. I'll change it to match the eslint version. I was trying to get the flat config working before switching to 9. Some of the plugins are very old and don't work so well. 

I have also been enabling the `unicorn` recommended rules to help covering the dropped plugins.